2024-03-01 11:44:19.186 [info] (Client) Pylance async client (2024.2.2) started with python extension (2024.2.1)
2024-03-01 11:44:22.844 [info] [Info  - 11:44:22] (48671) Pylance language server 2024.2.2 (pyright version 1.1.348, commit cfb1de0c) starting
2024-03-01 11:44:22.844 [info] [Info  - 11:44:22] (48671) Server root directory: file:///home/daniel/.vscode/extensions/ms-python.vscode-pylance-2024.2.2/dist
2024-03-01 11:44:22.856 [info] [Info  - 11:44:22] (48671) Starting service instance "Retos"
2024-03-01 11:44:23.069 [info] [Info  - 11:44:23] (48671) Setting pythonPath for service "Retos": "/home/daniel/Documentos/repos/django-tutorial-castellano/env/bin/python"
2024-03-01 11:44:23.071 [info] [Info  - 11:44:23] (48671) Setting environmentName for service "Retos": "3.11.2 (env venv)"
2024-03-01 11:44:23.227 [info] [Info  - 11:44:23] (48671) Assuming Python version 3.11
2024-03-01 11:44:26.440 [info] [Info  - 11:44:26] (48671) Found 2684 source files
2024-03-01 11:44:26.477 [info] [Info  - 11:44:26] (48671) Background analysis(1) root directory: file:///home/daniel/.vscode/extensions/ms-python.vscode-pylance-2024.2.2/dist
2024-03-01 11:44:26.478 [info] [Info  - 11:44:26] (48671) Background analysis(1) started
2024-03-01 11:44:28.485 [info] [Info  - 11:44:28] (48671) Indexer background runner(2) root directory: file:///home/daniel/.vscode/extensions/ms-python.vscode-pylance-2024.2.2/dist (index)
2024-03-01 11:44:28.485 [info] [Info  - 11:44:28] (48671) Indexing(2) started
2024-03-01 11:44:30.404 [info] [Info  - 11:44:30] (48671) scanned(2) 33 files over 1 exec env
2024-03-01 11:44:33.207 [info] [Info  - 11:44:33] (48671) indexed(2) 23 files over 1 exec env
2024-03-01 11:44:33.466 [info] [Info  - 11:44:33] (48671) Indexing finished(2).
2024-03-01 11:44:40.881 [info] [Warn  - 11:44:40] (48671) Workspace indexing has hit its upper limit: 2000 files
